test/dlc_tests/container_tests/bin/security_checks.py

```diff
@@ -26,11 +26,16 @@ def check_that_cache_dir_is_removed(home_dir):
     if os.path.exists(cache_dir_path):
         content_of_cache_dir = [f for f in os.listdir(cache_dir_path)]
         LOGGER.info("Contents of cache directory: %s", content_of_cache_dir)
-        if len(content_of_cache_dir) > 1:
-            raise ValueError("cache dir includes more than 1 file (not only motd)")
-        if not content_of_cache_dir[0].startswith("pip"):
-            raise ValueError("cache dir include file that it probably should not have: {}"
-                             .format(content_of_cache_dir[0]))
+        if len(content_of_cache_dir) == 2 and sorted(content_of_cache_dir) == ['mim', 'pip']:
```

Comment:
will this allow these for all containers, not just AG? is there a way to make an AG specific exception so that we do not add this rule everywhere?

Reply:
@gradientsky comment?

Reply:
Current codebase doesn't have an easy way to differentiate the logic based on framework.